src/core/views.py

```diff
@@ -2259,10 +2258,12 @@ def assign_proposal(request):
                     ]
 
             for field in data_fields:
-                if field.element.name in request.POST:
-                    # TODO change value from string to list [value, value_type].
-                    save_dict[field.element.name] = [
-                        request.POST.get(field.element.name),
+                field_name = field.element.name.encode(
```

Comment:
You're doing this over and over throughout this process. Wouldn't it be useful to create a single method that does this, limiting the possibility of getting `.encode('ascii', 'xmlcharrefreplace')` wrong with any of the points it's used?